content/guides/use-case/jupyter/customize.md

```diff
@@ -0,0 +1,261 @@
+
+1. In the **Launcher**, under **Notebook**, select **Python 3**.
+
+2. In the notebook, specify the following code.
+
+   ```python
+   from sklearn import datasets
+
+   iris = datasets.load_iris()
+   import matplotlib.pyplot as plt
+
+   _, ax = plt.subplots()
+   scatter = ax.scatter(iris.data[:, 0], iris.data[:, 1], c=iris.target)
+   ax.set(xlabel=iris.feature_names[0], ylabel=iris.feature_names[1])
+   _ = ax.legend(
+      scatter.legend_elements()[0], iris.target_names, loc="lower right", title="Classes"
+   )
+   ```
+
+3. Select the play button to run the code. You should see a scatter plot of the Iris dataset.
+
+In the terminal, press `ctrl`+ `c` to stop the container.
+
+## Use Compose to run your container
+
+Docker Compose is a tool for defining and running multi-container applications.
+In this case, the application isn't a multi-container application, but Docker
+Compose can make it easier to run by defining all the `docker run` options in a
+file.
+
+### Create a Compose file
+
+To use Compose, you need a `compose.yaml` file. In the same directory as your
+`Dockerfile`, create a new file named `compose.yaml`.
+
+Open the `compose.yaml` file in an IDE or text editor and add the following
+contents.
+
+```yaml
+services:
+  jupyter:
+    build:
+      context: .
+    ports:
+      - 8888:8888
+    volumes:
+      - .:/home/jovyan/work
+    command: start-notebook.py --NotebookApp.token='my-token'
+```
+
+The following is a line-by-line breakdown of the `compose.yaml` file.
+
+```yaml
+services:
+```
+
+This is the root key in the Docker Compose file, which you use to define the
+services that make up your application.
+
+```yaml
+  jupyter:
+```
+
+This is the name of the service you're defining. You can name your services
+whatever you like, but it's good practice to choose names that reflect the
+purpose of the service.
+
+```yaml
+    build:
+      context: .
+```
+
+`build` indicates that the service needs to be built from a Dockerfile and `context: .` specifies that the build context is the current directory (.).
+
+```yaml
+    ports:
+      - 8888:8888
+```
+You use `ports` to map ports between the host and the container. This maps port `8888` on the host to port `8888` on the container.
+
+```yaml
+    volumes:
+      - .:/home/jovyan/work
+```
+
+You use `volumes` to define mount points for volumes, which are directories that
+persist data across container restarts and allow for data sharing between the
+host and the container. In this case, `.:/home/jovyan/work` mounts the current
+directory `.` (where the Compose file is located) to `/home/jovyan/work` in the
+container.
+
+```yaml
+    command: start-notebook.py --NotebookApp.token='my-token'
+```
+
+You use `command` to run the `start-notebook.py --NotebookApp.token='my-token'`
+command when the container starts. This is for your convenience. Otherwise, the
+server starts with a random token that you must obtain from the logs.
```

Comment:
suggestion: again maybe we don't need to go into this level of detail when describing the compose file here?